classes/submission/Collector.php

```diff
@@ -156,6 +157,19 @@ public function filterByHasDois(?bool $hasDois, ?array $enabledDoiTypes = null):
         return $this;
     }
 
+    /**
+     * Limit results to submissions that should be listed on the DOI management page
+     *
+     * @param array|null $enabledDoiTypes TYPE_* constants to consider when checking submission has DOIs
+     */
+    public function filterByOnDoiPage(?bool $onDoiPage, ?array $enabledDoiTypes = null): AppCollector
+    {
+        $this->onDoiPage = $onDoiPage;
+        $this->enabledDoiTypes = $enabledDoiTypes === null ? [Repo::doi()::TYPE_PUBLICATION] : $enabledDoiTypes;
```

Comment:
If there are no enabled DOI types, should we not default to not showing them at all? Or does this break the logic somehow?